cmd/lambda/update_lease_status/main.go

```diff
@@ -65,10 +73,12 @@ func main() {
 			snsSvc:                                 &common.SNS{Client: sns.New(awsSession)},
 			leaseLockedTopicArn:                    common.RequireEnv("LEASE_LOCKED_TOPIC_ARN"),
 			emailSvc:                               &email.SESEmailService{SES: ses.New(awsSession)},
+			s3Svc:                                  s3Svc,
 			budgetNotificationFromEmail:            common.RequireEnv("BUDGET_NOTIFICATION_FROM_EMAIL"),
 			budgetNotificationBCCEmails:            common.RequireEnvStringSlice("BUDGET_NOTIFICATION_BCC_EMAILS", ","),
-			budgetNotificationTemplateHTML:         common.RequireEnv("BUDGET_NOTIFICATION_TEMPLATE_HTML"),
-			budgetNotificationTemplateText:         common.RequireEnv("BUDGET_NOTIFICATION_TEMPLATE_TEXT"),
+			budgetNotificationTemplatesBucket:      common.RequireEnv("BUDGET_NOTIFICATION_TEMPLATES_BUCKET"),
```

Comment:
Main change is the env var configuration here -- we're pointing at S3 objects, instead of embedding the templates in-full as env vars. We then just need to load the templates from S3 before sending emails.